contrib/docker/comdb2-docker-dev

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I think this can be wrapped into a [Docker Compose](https://docs.docker.com/compose/gettingstarted/) file in the future